plugins/GSdx/GSDevice9.cpp

```diff
@@ -188,6 +185,9 @@ void GSDevice9::ForceValidMsaaConfig()
 
 bool GSDevice9::Create(GSWnd* wnd)
 {
+	D3DTEXTUREFILTERTYPE LinearToAnisotropic = theApp.GetConfig("MaxAnisotropy", 0) && !theApp.GetConfig("paltex", 0) ? D3DTEXF_ANISOTROPIC : D3DTEXF_LINEAR;
+        D3DTEXTUREFILTERTYPE PointToAnisotropic = theApp.GetConfig("MaxAnisotropy", 0) && !theApp.GetConfig("paltex", 0) ? D3DTEXF_ANISOTROPIC : D3DTEXF_POINT;
+        
```

Comment:
Don't add whitespace issues, you've used tabs and spaces inconsistently and have trailing whitespace. Move the lines to just before the variables are used, somewhere after line 300.